.eslintrc.js

```diff
@@ -1,58 +1,29 @@
-
 module.exports = {
-  "extends": ["eslint:recommended", "plugin:react/recommended"],
+  extends: ['eslint:recommended', 'plugin:react/recommended', 'prettier'],
   env: {
     browser: true,
     node: true,
     es6: true,
     jasmine: true,
   },
   globals: {
-    "jest": true,
-    "shallowRender": true,
+    jest: true,
+    shallowRender: true,
   },
-  parser: "babel-eslint",
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "sourceType": "module",
-    "ecmaFeatures": {
-      "jsx": true
-    }
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
   },
-  plugins: [
-    "react",
-  ],
-  "rules": {
-    "no-console": 0,
-    "no-shadow": 2,
-    "no-var": 2,
-    // style
```

Comment:
No need for style rules — we're using Prettier so can rely on its own rules set. The `'prettier/prettier': 'error',` ensures linting will fail if there's contradiction with prettier's formatting.